packages/e2e-tests/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "2.0.1-alpha.1",
   "description": "For running end to end tests we use  Selenium WebDriver (https://www.selenium.dev/documentation/webdriver/) with Electron ChromeDriver (https://www.npmjs.com/package/electron-chromedriver) in Jest",
   "main": "index.js",
+  "private": true,
```

Comment:
How does private affect things? Does this prevent Lerna from bumping versions or just from publishing?

Reply:
It doesn't do anything until you apply `--no-private` flag to the `publish` or `version` command. I think it's best to just keep those "internal" packages marked as private but to continue to version them as we already do